README.md

```diff
@@ -75,13 +53,250 @@ We're pulling data from similiar APIs(All locations V2 - https://raw.githubuserc
 #### [Back end](🚫link to back end repo here) built using:
 
 #### 🚫 back end framework goes here (link to BE REPO)
-
-- point one
-- point two
-- point three
-
-🚫 List the rest of the back end end features and libraries in the same format as the framework above
-
+### dependencies
```

Comment:
The dependency section shouldn't be a copy paste from the package.json. Anyone who understands that file will go look in it if they want. This is a place to list major dependencies like Express, Axios, etc. I'd skip any mention of dev dependencies altogether.

Reply:
I removed dev dependencies